CHANGELOG.md

```diff
@@ -37,6 +40,11 @@ for objects that could not be processed
 - Updated async operations to handle writing records to the databases
 when output of the operation is `undefined`
 
+- **CUMULUS-2575**
```

Comment:
We need to note the changes to `get` and `exists` on the PG granule model. They're great changes, we just need to advertise them

Also, the change to `deconstructCollectionId` in the `message` package

And the new `createGranule` helper in the `api-client` package

Reply:
8850faba9a